fuel-client/src/client/schema/block.rs

```diff
@@ -72,6 +73,7 @@ pub struct BlockEdge {
 pub struct Block {
     pub id: BlockId,
     pub header: Header,
+    pub consensus: Option<Consensus>,
```

Comment:
Is there a reason an Option was chosen for Consensus instead of just having a default enum type in `Consensus` like `LocalAuthority` or similar?

Reply:
I thought that we can have not finished blocks without seal information. But if we need to have this kind of block in the future, I think we will use a separate type and endpoint. Replaced with `Consensus`